packages/abstractions/kiota_abstractions/serialization/serialization_writer.py

```diff
@@ -157,7 +157,7 @@ def write_bytes_value(self, key: Optional[str], value: bytes) -> None:
 
     @abstractmethod
     def write_object_value(
-        self, key: Optional[str], value: U, additional_values_to_merge: Optional[List[U]]
+        self, key: Optional[str], value: U, *additional_values_to_merge: Optional[List[U]]
```

Comment:
I believe the value should be optional in case we need to write None/null. Fixing it here and running the linting should probably address all my other comments in implementations.